### ISSUE TITLE
vb.metatag not working on Apple silicon

### ISSUE THREAD

**Issue by ElliotEtag** [v7b1/vb-objects#2]
Any plan to update this external, it is not compatible with Apple silicon (I use it in a M4L device to write audio clips' warped bpm to their file metadata) ?
Best

**Comment by v7b1** [comment]
Please try this: [https://vboehm.net/xchange/vb.metatag.zip](https://vboehm.net/xchange/vb.metatag.zip). No idea if this also works for M4L, though.

**Comment by ElliotEtag** [comment]
This is the one I was using i think. Unfortunately it doesn't load with Max running natively on Apple silicon processors (it works well running Max with Rosetta). I could not find any external doing this beside yours, I will miss it a lot
<img width="381" alt="Screenshot 2022-03-17 at 12 33 36" src="https://user-images.githubusercontent.com/36901823/158800646-050676a8-38f1-4c07-8038-03b45f1b409e.png">

**Comment by v7b1** [comment]
No, it's a newly compiled version, which runs on arm. Make sure to move the older one out of the way. the max window should show taglib version 1.12. 
I do have trouble getting tag.framework to link correctly in Monterrey, but the error you post has nothing to do with that.

**Comment by ElliotEtag** [comment]
Oh i am sorry. 
Now i think it loaded the right one. So this is the Monterrey issue then?
<img width="804" alt="Screenshot 2022-03-17 at 13 04 12" src="https://user-images.githubusercontent.com/36901823/158804767-ecb2bb60-ec32-4ed6-8e01-6ac7f4f31079.png">

**Comment by v7b1** [comment]
yes, exactly. So obviously it doesn't work with a relative path, too bad. 
Can you please try to copy or move the tag.framework found in /source/libs/taglib-build to /Library/Frameworks/ folder?

**Comment by ElliotEtag** [comment]
yes ! Amazing thank you!
<img width="136" alt="Screenshot 2022-03-17 at 13 16 11" src="https://user-images.githubusercontent.com/36901823/158806580-646868f8-dc50-4845-8397-6796fe73dea5.png">

**Comment by sorrenti90** [comment]
Hey there, apologies for revisiting this problem.

I'm currently utilizing vb.metatag within an M4L device to extract metadata from audio clips, specifically fetching the cover art, artist, and title. However, I've encountered a small hiccup. When the Max editor is open, vb.metatag functions smoothly, generating coverart.jpg in the /tmp folder and extracting artist and title information as expected. However, when I use it as an M4L device (with the Max editor closed), vb.metatag fails to work. It doesn't generate coverart.jpg nor does it extract artist/title information.

I'm unsure whether I've made an error in the patch programming or if there's a specific setting within M4L that I'm overlooking. 
I share with you two my M4L device. 
Any insights would be greatly appreciated!

[vbmetatag_issue.amxd.zip](https://github.com/v7b1/vb-objects/files/14980360/vbmetatag_issue.amxd.zip)
